src/test/txpackage_tests.cpp

```diff
@@ -368,6 +370,94 @@ BOOST_FIXTURE_TEST_CASE(package_submission_tests, TestChain100Setup)
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(tx_parent->GetHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(tx_child->GetHash())));
     }
+
+    // do not allow parents to pay for children
+    {
+        Package package_ppfc;
```

Comment:
Lots of magic numbers in this test. Could you assert specific vsizes on each tx, and have that number match the picture here?

Reply:
Done in #26711 (https://github.com/bitcoin/bitcoin/compare/0cdb25a2392328afaad7651a0a407396b1922f17..f4e449254e41172b09a41ed9b2cc5c5fe3a5edca)